From 3ef6d120e3e85d6ad12e279b1158633d7edd9205 Mon Sep 17 00:00:00 2001
From: lennard <44114474+Dummerle@users.noreply.github.com>
Date: Thu, 5 Jan 2023 22:07:05 +0100
Subject: [PATCH] Update to legendary 0.20.32

---
 pyproject.toml   | 2 +-
 requirements.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 20637c149..61a2250a8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,7 +30,7 @@ pywebview = [
     { version = "^3.6.3", extras = ["cef"], platform = "windows", optional = true },
     { version = "^3.6.3", extras = ["gtk"], platform = "linux", optional = true },
 ]
-legendary-gl = "^0.20.31"
+legendary-gl = "^0.20.32"
 typing-extensions = "^4.3.0"
 
 [tool.poetry.scripts]
diff --git a/requirements.txt b/requirements.txt
index 938ce6fa2..51dbd5345 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -3,5 +3,5 @@ requests
 PyQt5
 QtAwesome
 setuptools
-legendary-gl==0.20.31
+legendary-gl==0.20.32
 pywin32; platform_system == "Windows"
